### PR TITLE
chore: api examples to es6

### PR DESCRIPTION
**Pull request by jajugoguma**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
- Change examples in api document(jsdoc) to ES6
- Prettier table of contents in README
- Fix the version of the master branch because it does not match the last version that was actually deployed due to an error

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/pagination.js [hunk 1]
```diff
@@ -337,7 +339,7 @@ var Pagination = defineClass(
        * @property {number} page - Moved page
        * @example
        * paganation.on('beforeMove', function(evt) {
```
Comment (lja1018) [inline src/js/pagination.js]: 화살표함수로 바꾸는게 어떤가요?

File: src/js/pagination.js [hunk 2]
```diff
@@ -357,7 +359,7 @@ var Pagination = defineClass(
        * @property {number} page - Moved page
        * @example
        * paganation.on('afterMove', function(evt) {
```
Comment (lja1018) [inline src/js/pagination.js]: 여기도요!

**Review by lja1018 — COMMENTED** [review]
리뷰완료합니다.

File: src/js/pagination.js [hunk 3]
```diff
@@ -43,10 +43,12 @@ var defaultOption = {
  *     @param {boolean} [options.usageStatistics=true] Send the hostname to google analytics.
  *         If you do not want to send the hostname, this option set to false.
  * @example
- * var Pagination = tui.Pagination; // or require('tui-pagination')
+ * import Pagination from 'tui-pagination'; // ES6
+ * // const Pagination = require('tui-pagination'); // CommonJS
+ * // const Pagination = tui.Pagination;
```
Comment (jungeun-cho) [inline src/js/pagination.js]: 주석처리할 필요가 있을까요?
네임스페이스 사용하는건 `// Browser` 코멘트 추가하는게 어때요?

**Review by jungeun-cho — APPROVED** [review]
목차 윗내용 제거랑 네임스페이스 사용하는거 Browser 표기 확인해주세요~

**Pull request merged**